docs/getting-started/agent-tamper-protection.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[agent-tamper-protection]]
+= Prevent {agent} uninstallation
+
+For hosts enrolled in {elastic-defend}, you can prevent unauthorized attempts to uninstall {agent} and {elastic-endpoint} by enabling *Agent tamper protection* on the Agent policy. This offers an additional layer of security by preventing users from bypassing or disabling {elastic-defend}'s endpoint protections. 
+
+When enabled, {agent} and {elastic-endpoint} can only be uninstalled on the host by including an uninstall token in the uninstall CLI command. One unique uninstall token is generated per Agent policy, and you can retrieve uninstall tokens in an Agent policy's settings or in the {fleet} UI.
+
+.Requirements
```

Comment:
We may want to add one more call out here that Agents must be at `8.11.0` or higher to use this feature.  It is possible for users to have their stack upgraded, but they haven't upgraded their Agents yet.

Reply:
Thanks! Done in https://github.com/elastic/security-docs/pull/4232/commits/5c1324a24f1cbc36b54fa1740b268ca4ac48f415.

Reply:
thanks!